docs/wiki/research/PBS/ePBS.md

```diff
@@ -0,0 +1,619 @@
+
+- **Permissionlessness and Censorship Resistance** While TBHL aims to maintain a permissionless environment and combat censorship, achieving these goals within the new framework poses challenges. Ensuring that any builder can submit bids and that proposer blocks fairly represent the competitive landscape requires transparent and secure handling of the bidpool. Moreover, integrating forward inclusion lists or other censorship resistance mechanisms within the TBHL structure necessitates additional protocol considerations to prevent manipulation or exclusionary practices.
+
+- **Compatibility with Existing and Future Ethereum Features** TBHL must seamlessly integrate with Ethereum's current protocol features and be adaptable to future innovations, such as single-slot finality (SSF) and MEV-burn mechanisms. Ensuring compatibility with these evolving aspects of the Ethereum ecosystem demands a forward-looking approach to design and implementation, capable of accommodating adjustments and enhancements without undermining the TBHL framework's integrity or efficacy.
+
+- **Resource and Computational Overheads** The introduction of TBHL introduces new computational and resource overheads, particularly related to handling the increased data volume from the dual-block mechanism and the additional attestation rounds. Optimizing the protocol to manage these demands efficiently, without significantly increasing the computational burden on validators or compromising the network's performance, is an essential engineering concern.
+
+The TBHL proposal stands as a testament to the ongoing efforts to refine Ethereum's PBS mechanisms, striving for a balance between operational efficiency, security, and the overarching ethos of decentralization. By addressing the nuances of proposer-builder dynamics and introducing robust safeguards, TBHL marks a significant step forward in the evolution of Ethereum's protocol design, offering a promising avenue for mitigating the challenges posed by MEV while enhancing the network's resilience and integrity.
+
+### Payload-Timeliness Committee (PTC)
+
+The Payload-Timeliness Committee (PTC) proposal is a design for enshrining Proposer-Builder Separation (ePBS) within the Ethereum protocol. It represents an evolution of the mechanism to determine block validity and includes a subset of validators who vote on the timeliness of a block's payload[^7].
+
+#### High-Level Overview
+
+```mermaid
+sequenceDiagram
+    autonumber
+    rect rgb(240, 237, 225)
```

Comment:
What's the purpose of this? Doesn't seem readable

Reply:
Just like code blocks in GitHub, this will autogenerate a sequence flow diagram when the GitHub renders the page. The neat feature is GitHub will make this sequence digram interactive for the user with some controls for zoom in/out, pan left/right/up/down. For example, this code block will render on the page like this. 

<img width="547" alt="image" src="https://github.com/eth-protocol-fellows/protocol-studies/assets/3957491/68f3810a-b2e7-4555-be19-49107d093572">

Reply:
Neat!